docs/architecture.md

```diff
@@ -1,20 +1,20 @@
 # Architecture
 
-The diagram presents the basic workflow of Kyma Showcase. <!--1. the numbers could be also presented in the diagram, 2. don't capitalize "event">
+The diagram presents the basic workflow of Kyma Showcase.
 
 ![Diagram Kyma Showcase](./assets/diagram_showcase.svg)
 
 1. Frontend gets all images stored in the Redis Database and displays them in the feed.
 2. A user uploads an image using Frontend's UI.
-3. The image is stored in the database and then displayed in the feed. <!--the step is similar to the first one>
-4. Backend sends an event containing the database ID used to access the image.
-5. Serverless Functions are triggered by the event and get the image from the database using the given ID.
+3. The newly uploaded image is stored in the Redis Database and then displayed in the feed.
+4. Backend sends an event containing the Redis Database ID used to access the image.
+5. Serverless Functions are triggered by the event and get the image from the Redis Database using the given ID.
 6. The Functions send the image in base64 to Google Cloud Platform for processing.
```

Comment:
```suggestion
6. The Functions send the image in base64 to the Google Cloud Platform (GCP) for processing.
```

This is the first time you use "Google Cloud Platform". Let's add the abbreviation in brackets here and use GCP from now on in the doc.